mappings/net/minecraft/world/biome/BiomeGenerator.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS none/ajk net/minecraft/world/biome/BiomeGenerator
-	FIELD a isDecorating Z
+	FIELD a decorating Z
```

Comment:
Consistensy is more important to me personally. Don't want to end up with some fields named like this and others like that, same with methods.